tests/test-sessions/test-infinispan-sessions/src/test/java/org/eclipse/jetty/server/session/remote/RemoteInfinispanTestSupport.java

```diff
@@ -138,6 +76,67 @@ public RemoteInfinispanTestSupport(String cacheName)
             cacheName = DEFAULT_CACHE_NAME + System.currentTimeMillis();
 
         _name = cacheName;
+
+        if (!infinispan.isRunning())
+        {
+            try
+            {
+                long start = System.currentTimeMillis();
+                String infinispanVersion = System.getProperty("infinispan.docker.image.version", "11.0.9.Final");
+
+                infinispan.start();
+                String host = infinispan.getContainerIpAddress();
+                System.setProperty("hotrod.host", host);
+                int port = infinispan.getMappedPort(11222);
+
+                LOG.info("Infinispan container started for {}:{} - {}ms", host, port,
+                        System.currentTimeMillis() - start);
+                SearchMapping mapping = new SearchMapping();
+                mapping.entity(InfinispanSessionData.class).indexed().providedId()
+                        .property("expiry", ElementType.METHOD).field();
+
+                Properties properties = new Properties();
+                properties.put(Environment.MODEL_MAPPING, mapping);
+
+                ConfigurationBuilder configurationBuilder = new ConfigurationBuilder().withProperties(properties)
+                        .addServer().host(host).port(port)
+                        // we just want to limit connectivity to list of host:port we knows at start
+                        // as infinispan create new host:port dynamically but due to how docker expose host/port we cannot do that
+                        .clientIntelligence(ClientIntelligence.BASIC)
+                        .marshaller(new ProtoStreamMarshaller());
+
+                if (infinispanVersion.startsWith("1"))
+                {
+                    configurationBuilder.security().authentication()
+                            .saslMechanism("DIGEST-MD5")
+                            .username("theuser").password("foobar");
+                }
+
+                configurationBuilder.addContextInitializer(new InfinispanSerializationContextInitializer());
+                Configuration configuration = configurationBuilder.build();
+
+                _manager = new RemoteCacheManager(configuration);
```

Comment:
`_manager` is declared as `static` but is referenced here as an instance variable.

Reply:
not static anymore